app/javascripts/main/menuManager.js

```diff
@@ -23,6 +23,16 @@ class MenuManager {
     window.setMenuBarVisibility(isMenuBarVisible);
 
     const template = [
+      {
+        label: 'App',
```

Comment:
Would you mind removing the label property and instead use `role: 'filemenu'`? I can do it myself if you don't have the project handy anymore 🙂

Reply:
I'll change it this or tomorrow evening. If you want to change it before I find the time go ahead :)